prysm/helm/templates/validator-deployment.yaml

```
@@ -32,41 +32,18 @@ spec:
         - --{{ $.Values.ethereumTestnet }}
         {{- end }}
         - --beacon-rpc-provider={{ $.Values.beacon.name }}-service.{{ $.Values.namespace }}.svc.cluster.local:4000
-        - --datadir=/data/prysm/validator-client
-        - --wallet-dir=/data/prysm/wallet
+        - --datadir={{ $validatorClient.dataVolumePath }}
+        - --wallet-dir={{ $validatorClient.walletVolumePath }}
         - --wallet-password-file=/data/prysm/password/wallet-pwd.txt
         - --graffiti={{ $validatorClient.graffiti | quote }}
         - --accept-terms-of-use
         volumeMounts:
-        - name: validator-client-storage
-          mountPath: /data/prysm/validator-client
-          readOnly: false
-        - name: wallet-storage
-          mountPath: /data/prysm/wallet
-          readOnly: false
+        {{- include "prysm.volumeMounts" (merge $validatorClient (pick $.Values "persistentVolumeType" "nfs")) | indent 8 }}

```

```suggestion
        {{- include "prysm.volumeMounts" $validatorClient | indent 8 }}

```

It seems that persistentVolumeType and nfs are not used in "prysm.volumeMounts" and "unique.volumes" tpls.